lib/rules/disallow-padding-newlines-after-blocks.js

```diff
@@ -0,0 +1,104 @@
+/**
+ * Disallow a newline after blocks
+ *
+ * Type: `Boolean`
+ *
+ * Values: `true`
+ *
+ * #### Example
+ *
+ * ```js
+ * "disallowPaddingNewLinesAfterBlocks": true
+ * ```
+ *
+ * ##### Valid
+ *
+ * ```js
+  * function () {
+ *     for (var i = 0; i < 2; i++) {
+ *         if (true) {
+ *             return false;
+ *         }
+ *         continue;
+ *     }
+ *     var obj = {
+ *         foo: function() {
+ *             return 1;
+ *         },
+ *         bar: function() {
+ *             return 2;
+ *         }
+ *     };
+ * }
+ * ```
+ *
+ * ##### Invalid
+ *
+ * ```js
+ * function () {
+ *     for (var i = 0; i < 2; i++) {
+ *         if (true) {
+ *             return false;
+ *         }
+ *
+ *         continue;
+ *     }
+ *
+ *     var obj = {
+ *         foo: function() {
+ *             return 1;
+ *         },
+ *
+ *         bar: function() {
+ *             return 2;
+ *         }
+ *     };
+ * }
+ * ```
+ */
+
+var assert = require('assert');
+
+module.exports = function() {};
+
+module.exports.prototype = {
+
+    configure: function(value) {
+        assert(
+            typeof value === 'boolean',
+            'disallowPaddingNewLinesAfterBlocks option requires boolean value'
+        );
+        assert(
```

Comment:
Definitely. Just both assertions will throw if `value !== true`, and it's only possible value for rule. And also `option requires boolean value` can mislead. So why we need to assert twice? ;-) That's why we started that discussion earlier.